modules/virt-installing-qemu-guest-agent-on-linux-vm.adoc

```diff
@@ -36,3 +33,7 @@ $ yum install -y qemu-guest-agent
 ----
 $ systemctl enable --now qemu-guest-agent
 ----
+
+.Verification
+
+To check if your VM has the QEMU guest agent installed and running, verify that `AgentConnected` is listed in the VM spec.
```

Comment:
It might be helpful to have a command that would help them verify the `AgentConnected` is listed in the VM spec.

Reply:
@sabrinajess 
The command would be `$ oc get vm <vm_name>`. The [output](https://kubevirt.io/user-guide/virtual_machines/guest_agent_information/) should look like this:

status:
  conditions:
  - lastProbeTime: "2020-02-28T10:22:59Z"
    lastTransitionTime: null
    status: "True"
    type: AgentConnected